CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # IdentityCache changelog
 
+#### 0.5.0
+
+- Deprecated embedded associations on models that don't use IDC (#305)
+- Remove a respond_to? check that hides mistakes in includes hash (#307)
+- refactor: Include modules into IdentityCache instead into models (#303)
```

Comment:
Refactoring shouldn't matter to the user's of the library.  Only include changes that affect the users of the library